configs/config.dev.yaml

```diff
@@ -194,6 +194,12 @@ oncall-key: abc # API key for oncall auth
 applications:
  - dummy_app
 
+## Enable addition of metavariables to the incident context by default.
+## If disabled, the metavariables will only added in the incident context for plans in the triage_allow_list.
+enable_default_metavariables_in_context: False
+triage_allow_list:
```

Comment:
nit: since we renamed the config lets also rename the allow list maybe (metavariables_in_context_allow_list) or something. Otherwise everything else lgtm